src/test/java/com/wikia/webdriver/testcases/mobilewikitests/curatedcontenttests/EditorTests.java

```diff
@@ -45,7 +45,7 @@ public void MercuryCuratedEditorTest_001_addAndSaveItemToFeaturedContent() {
     Boolean result = new ArticlePage().open(MercurySubpages.ECC_MAIN_PAGE).getCuratedMainPage()
         .isCuratedElementVisible(FEATURED_CONTENT_SELECTOR);
 
-    PageObjectLogging.logInfo(String.format("Curated content is visible: %s", result));
+    Log.info(String.format("Curated content is visible: %s", result));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Curated content is visible: %s" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)